Multisig/Features/Connect to Web/Logic/WebConnectionRepository.swift

```diff
@@ -14,6 +14,19 @@ class WebConnectionRepository {
     private static let REQUEST_ID_DOUBLE_NIL: Double = -1
     private static let ICONS_SEPARATOR: String = "|"
 
+    func connections() -> [WebConnection] {
+        var connections: [WebConnection] = []
+        do {
+            let nillableConnectionList = try CDWCConnection.getAll().map { connection(from: $0) }.filter { $0 != nil }
```

Comment:
let result = ...getAll().compactMap { connection(from: $0) } - this would replace these 4 lines completely.